t/api/10-jobgroups.t

```diff
@@ -479,4 +479,91 @@ subtest 'prevent create/update duplicate job group on same parent group' => sub(
     );
 };
 
+subtest 'update default_keep_logs_in_days and default_keep_results_in_days' => sub() {
+    # TODO: refactor this test by extracting the common code from the next test
+    # parent group test
+    $t->post_ok(
+        '/api/v1/parent_groups',
+        form => {
+            name => 'test_parent_group',
+            default_keep_logs_in_days => 888,
+            default_keep_results_in_days => 777,
+        }
+    )->status_is(400, 'if default_keep_logs_in_days greater than default_keep_results_in_days create should fail')
+     ->json_is(
+        '/error' => '`default_keep_logs_in_days` must be grater than `default_keep_results_in_days`',
```

Comment:
Not sure about the choice of
<p><a href="https://commons.wikimedia.org/wiki/File:Kuechenreibe_(fcm).jpg#/media/File:Kuechenreibe_(fcm).jpg"><img src="https://upload.wikimedia.org/wikipedia/commons/4/43/Kuechenreibe_%28fcm%29.jpg" alt="Kuechenreibe (fcm).jpg" height="480" width="624"></a><br>By Frank C. Müller, <a href="https://creativecommons.org/licenses/by-sa/4.0" title="Creative Commons Attribution-Share Alike 4.0">CC BY-SA 4.0</a>, <a href="https://commons.wikimedia.org/w/index.php?curid=374762">Link</a></p>
here :)

Reply:
This takes me a while 🤣